backends/arm/test/ops/test_sigmoid.py

```diff
@@ -122,6 +128,9 @@ def _test_sigmoid_tosa_ethos_BI_pipeline(
             .to_executorch()
         )
 
+        if conftest.is_option_enabled("tosa_ref_model"):
+            tester.run_method_and_compare_outputs(inputs=test_data)
```

Comment:
This is not right, we don't want to change the way we do things on U55/U85 just for tosa-ref-model.